Tests/SuperLinq.Test/BatchTest.cs

```diff
@@ -25,18 +25,18 @@ public void BatchValidatesSize()
 
 		_ = Assert.Throws<ArgumentOutOfRangeException>("size",
```

Comment:
@viceroypenguin In `BatchTests`:

- `BatchValidatesSize` validates the exceptions thrown when `size` is invalid. Should we do the same for the other arguments?